lib/models/builder.js

```diff
@@ -18,8 +20,22 @@ module.exports = Task.extend({
     process.env.EMBER_ENV = process.env.EMBER_ENV || this.environment;
 
     var broccoli = require('broccoli');
-    this.tree    = broccoli.loadBrocfile();
+    var tree = broccoli.loadBrocfile();
```

Comment:
I don't believe we don't need to call it, unless `ember-cli-build.js` isn't present